war/src/main/less/modules/buttons.less

```diff
@@ -123,7 +123,7 @@ a.yui-button:visited {
   box-sizing: border-box;
   // vertical padding:
   // 0.375rem == 6px == 32px (target height) - 4px (borders) - 16 (line) / 2
-  padding: 0.375rem 1rem;
+  padding: 0.375rem 0.5rem;
```

Comment:
I think this change should not take place because it will affect all buttons and it can be have unexpected . Any specific modifications could be done using custom CSS.